### ISSUE TITLE
SOI images low res. Out of date.

### ISSUE THREAD

**Issue by lodici** [magarena/magarena#1144]
Most of my Shadows over Innistrad images are low-res but when following the link there is a better quality image available. Is it just me?

**Comment by LoubiTek** [comment]
I do not think so no, it's real !
https://github.com/magarena/magarena/blob/master/resources/cardbuilder/frames/transform/planeswalker/ccard.jpg
Hum I am not sure..?

**Comment by ShawnieBoy** [comment]
This will be an issue when magiccards.info update their images, but we haven't updated the 'update date' for cards from that set which haven't been reprinted since. Cardbuilder will not do this, although it will use the correct printing url - it has no information on when a set was last downloaded.

I remember something about a 60 day window from download and update date - or something. If a user has a last downloaded date 60 days before the update date on the scripts.

Just to add - my local files are high res, so I don't know when you last downloaded them, or if the magiccards.info files have actually changed at all.

**Comment by ShawnieBoy** [comment]
> Hum I am not sure..?

As far as cardbuilder is concerned, crops do not matter so much for resolution. They can still be read. People who use the full card image will have more difficulty reading card text.

**Comment by lodici** [comment]
> I remember something about a 60 day window from download and update date - or something. If a user has a last downloaded date 60 days before the update date on the scripts.

Yes, that could be it. I will need to check the code to be sure.

**Comment by lodici** [comment]
The dates are reset back by -60 days during the [import](https://github.com/magarena/magarena/blob/master/src/magic/ui/ImportWorker.java#L249) -
```
        // override download dates to catch any missed "image_updated" updates.
        final Calendar dt = Calendar.getInstance();
        dt.add(Calendar.DAY_OF_MONTH, -60);
        CONFIG.setUnimplementedImagesDownloadDate(dt.getTime());
        CONFIG.setPlayableImagesDownloadDate(dt.getTime());
```
